activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+*   Don't allow that `where` with invalid value matches to nil values.
```

Comment:
I think we should remove `that` from here. 

Following sounds correct to me. 
```
Don't allow `where` with invalid value matches to nil values.
```

Reply:
Thank you for the pointing out, I've removed that.